src/tokens/saga.ts

```diff
@@ -253,25 +251,27 @@ interface UserBalancesResponse {
 export async function fetchTokenBalancesForAddress(
   address: string
 ): Promise<FetchedTokenBalance[]> {
-  const response = await apolloClient.query<UserBalancesResponse, { address: string }>({
-    query: gql`
-      query FetchUserBalances($address: Address!) {
-        userBalances(address: $address) {
-          balances {
-            tokenAddress
-            balance
-          }
-        }
-      }
-    `,
-    variables: {
-      address,
-    },
-    fetchPolicy: 'network-only',
-    errorPolicy: 'all',
-  })
-
-  return response.data.userBalances.balances
+  await sleep(100)
+  throw new Error('test')
```

Comment:
as with the other PR, this is for testing! in case you want to check out the branch and play with the error. i'm faking a refetch + fail after 1 s